mqtt/mqttd/src/broker/bootstrap/edgehub.rs

```diff
@@ -131,6 +138,12 @@ where
     pin_mut!(renewal_signal);
     let shutdown = future::select(shutdown_signal, renewal_signal).map(drop);
 
+    // TODO remove this call when broker readiness is implemented
+    tokio::spawn(async move {
```

Comment:
Is `broker_ready`  going to be passed into some `CommandHandler` to trigger the signal?